### ISSUE TITLE
Plugins Correct, HTTPs Error

### ISSUE THREAD

**Issue by TrinaryAtom** [kanboard/kanboard#4064]
I am trying to install plugins through the GUI and i know i followed all of the instructions correctly and to the T. However, it would not produce an error message or inform me in any way that something was not right.

I pushed the issue for about 6 hours, all while thinking: "Why would there be a plugins json list, just sitting out there, but its not used, why?"

Well, I spent the better part of that 6 hours before it hit me. Minimal install. Maybe since the location of these plugins were sitting behind SSL Certificate it was not validating against it and failing. So i install **php-curl** and **curl** in order to install **ca-certificates** (I realize i may have not needed the curl installs, but, that's what i did to solve my problem (plus i was going to use curl later anyway, for a separate application)).

---
TL;DR
-
Can you please update the [documentation on the admin plugins page](https://docs.kanboard.org/en/latest/admin_guide/plugins.html) to include the possibility of needing **ca-certificates**? Or have the plugins page output the certificate error. Might save some individuals some time who like to build minimal server installations. Thank you kindly!

**Comment by fguillot** [comment]
The [website](https://github.com/kanboard/website) is hosted on GitHub pages. GitHub allows only TLS 1.2 for security reasons (see https://blog.github.com/2018-02-23-weak-cryptographic-standards-removed/).

If running `curl https://kanboard.org/plugins.json` on your server doesn't work. That means you are probably using outdated packages/operating systems.

**Comment by TrinaryAtom** [comment]
Sorry for the confusion, I wasn't using curl. I stated I only installed it because it also installed ca-certificates. I just noted what i did to install it. I was going to use curl on the server for another application.

I will test without installing curl of any kind and report back.

**Comment by TrinaryAtom** [comment]
Confirmed: As suspected, on a minimal Debian 9 Install (and unchecked all options in tasksel on install) ca-certificates is not installed. Which causes kanboard to not produce any errors when attempting to retrieve any packages via the https://kanboard.org/plugins.json. Simply installing ca-certificates, fixes this problem.

Simply put, if someone was installing minimally, and they forgot about needing/installing ca-certificates, there are no errors produced. Which might confuse others in the future.
